### PR TITLE
Remove Datadog

### PR DESCRIPTION
**Pull request by jrusbatch**
Discord's built-in analytics have improved significantly in the past few years, to the point that reporting metrics to Datadog no longer feels worthwhile.

Last week I revoked the API key that we were using to report Datadog metrics from MODiX and its underlying infrastructure. This PR removes the Datadog NuGet packages from all projects that had been referencing it, as well as all logic related to reporting metrics to Datadog.

### CODE REVIEW

File: Modix/Modix.csproj [hunk 1]
```diff
@@ -65,78 +64,4 @@
     </Content>
   </ItemGroup>
 
-  <Target Name="EnsureNodeEnv" BeforeTargets="Build" Condition=" '$(Configuration)' == 'Debug' And !Exists('$(SpaRoot)node_modules') ">
```
Comment (Cisien) [inline Modix/Modix.csproj:68]: Is this change intended?

**Review by Cisien — APPROVED** [review]

**Review by Scott-Caldwell — APPROVED** [review]
